src/JuliusSweetland.OptiKey/UI/Views/Keyboards/Common/Menu.xaml

```diff
@@ -48,10 +48,9 @@
                       SharedSizeGroup="KeyWithSymbolAndText"
                       Value="{x:Static models:KeyValues.NumericAndSymbols1KeyboardKey}"/>
         <controls:Key Grid.Row="0" Grid.Column="2"
-                      SymbolGeometry="{StaticResource DiacriticsIcon}"
-                      Text="{x:Static resx:Resources.DIACRITICS}"
+                      Text="Dynamic!"
```

Comment:
Could you please revert for now

Reply:
I had another thought on this...

We could retain the current Menu keyboard, but also define exactly the same keyboard in xml as a dynamic keyboard called Menu.xml (or similar). If detected in the keyboard directory (defined in the setting) then this is used to render the menu, giving the end user control of how their dynamic keyboards are used and which keys they replace to get to their other dynamic keyboards. If not detected then OptiKey can fall back to the built in Menu.xaml keyboard.

What do you think?